### PR TITLE
feat(programming-language-support): Add dooku.nvim

### PR DESCRIPTION
**Pull request by Zeioth**
Adds [dooku.nvim](https://github.com/Zeioth/dooku.nvim) under `/comment`

### CODE REVIEW

**Comment by Zeioth** [comment]
Maybe it would be better to create the directory `code-documentation` instead of adding it to `comment`, but at the moment it would only have dooku.nvim, and I didn't want to add more complexity to the repo.

**Comment by Uzaaft** [comment]
@Zeioth  Next time please read the Contributing.md. :)

File: lua/astrocommunity/comment/dooku-nvim/init.lua
Comment (Uzaaft) [inline lua/astrocommunity/comment/dooku-nvim/init.lua:8]: This shouldn't be needed.

Comment (Zeioth) [reply lua/astrocommunity/comment/dooku-nvim/init.lua:5]: Fixed in https://github.com/AstroNvim/astrocommunity/pull/503/commits/35134589c4f3b3412ad31a6a1874eddcdf74c897

**Comment by Uzaaft** [comment]
@Zeioth  You misunderstood what I said. That config = function(_,opts)..... stuff should not be there, just the opts. Like this:
image

**Comment by Uzaaft** [comment]
Also, I think this plugin should be under programming-tools or something. Comment doesn't seem to fit it's category.

**Comment by Zeioth** [comment]
Ok @Uzaaft I'm moving it to `programming-tools`. But the config line is necessary. The plugin doesn't do that for you atm.

**Comment by Zeioth** [comment]
Done, please let me know if you want any other change.

**Comment by Uzaaft** [comment]
I still don't see the opts table there.

Here, I'll give you the code:
```lua
return {
  {
    "Zeioth/dooku.nvim",
    cmd = { "DookuGenerate", "DookuOpen", "DookuAutoSetup" },
	opts = {}
  }
}
```

**Review by Uzaaft — APPROVED** [review]

**Comment by Zeioth** [comment]
@Uzaaft thank you for your patience, I've tested the code, and now I see when passing opts = {}, the config line is not necessary.

**Comment by Uzaaft** [comment]
> @Uzaaft thank you for your patience, I've tested the code, and now I see when passing opts = {}, the config line is not necessary.

Sweet. :)

**Comment by Uzaaft** [comment]
Alright. I just double checked before I pressed the merge button. Sorry about all the back and forth @Zeioth.
Stylua is complaining about a missing ,. Fixed down below.  The group we should use is `programming-language-support`
```lua
return {
  {
    "Zeioth/dooku.nvim",
    cmd = { "DookuGenerate", "DookuOpen", "DookuAutoSetup" },
	opts = {},
  }
}
```
We'll merge if this is fixed. Again, sorry about it @Zeioth

**Comment by Zeioth** [comment]
@Uzaaft  woop, miss understanding on my side. changes applied on https://github.com/AstroNvim/astrocommunity/pull/503/commits/3bcaab8e9ea9cfe43fc976a2b52c36e26ca1d7c0

* comma added after opts = {},
* moved to directory `programming-language-support`

File: lua/astrocommunity/programming-language-support/dooku-nvim/init.lua
Comment (Uzaaft) [inline lua/astrocommunity/programming-language-support/dooku-nvim/init.lua:6]:
```suggestion
  },
```

**Review by Uzaaft — APPROVED** [review]

**Review by luxus — APPROVED** [review]

**Pull request merged**
